utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -38,7 +38,11 @@ class HtmlParser private constructor(
     }
 
     if (htmlContent.contains(CUSTOM_IMG_TAG)) {
-      htmlContent = htmlContent.replace(CUSTOM_IMG_TAG, REPLACE_IMG_TAG, /* ignoreCase= */false)
+      htmlContent = htmlContent.replace(
+        CUSTOM_IMG_TAG,
+        REPLACE_IMG_TAG,
+        /* ignoreCase= */ false
+      )
       htmlContent = htmlContent.replace(
         CUSTOM_IMG_FILE_PATH_ATTRIBUTE,
         REPLACE_IMG_FILE_PATH_ATTRIBUTE, /* ignoreCase= */false
```

Comment:
As discussed with other teammates in my other app module PR, this one comes. 
All should be in a separate line. 

```
htmlContent = htmlContent.replace(
        CUSTOM_IMG_FILE_PATH_ATTRIBUTE,
        REPLACE_IMG_FILE_PATH_ATTRIBUTE,
        /* ignoreCase= */ false
)
```

Reply:
Done